### PR TITLE
Feature/mc 17233 download object

### PR DESCRIPTION
**Pull request by AnKhangC**
#### Changes made
Screen Shot 2022-03-22 at 10 43 24 AM

#### Related PRs
- []()

### CODE REVIEW

File: source/includes/aws/_objects.md [hunk 1]
```diff
@@ -58,6 +57,30 @@ Attributes | &nbsp;
...
 
 
+<!-------------------- DOWNLOAD AN OBJECT -------------------->
+
+#### Download Object
+
+```shell
+curl --request GET \
+  -H "MC-Api-Key: your_api_key" \
+  "http://localhost:8081/rest/services/operations/:service_connection_id/download?bucketName=akc-buck&regionName=us-east-1&id=us-east-1/akc-buck/api123.png&entityType=objects&operation=download&environmentId=97430433-875c-4ba5-8c57-90a051a52729"
```
Comment (evanluc) [inline source/includes/aws/_objects.md]: can we replace the <localhost:8080> with <YOUR_CLOUDMC_ENDPOINT> or whatever the placeholder we use is usually set as.

Comment (AnKhangC) [reply source/includes/aws/_objects.md]: [44142aa](https://github.com/cloudops/cloudmc-api-docs/pull/483/commits/44142aace66c3f027eb67556128db04fd0fa8db2)

**Review by evanluc — APPROVED** [review]

File: source/includes/aws/_objects.md [hunk 2]
```diff
@@ -58,6 +57,30 @@ Attributes | &nbsp;
...
+#### Download Object
+
+```shell
+curl --request GET \
+  -H "MC-Api-Key: your_api_key" \
+  "https://cloudmc_endpoint/rest/services/operations/:service_connection_id/download?bucketName=akc-buck&regionName=us-east-1&id=us-east-1/akc-buck/api123.png&entityType=objects&operation=download&environmentId=97430433-875c-4ba5-8c57-90a051a52729"
+```
+
+<code>GET /services/operation/:service_connection_id/download</code>
+
```
Comment (lamminade) [inline source/includes/aws/_objects.md:73]: should probably add a line explaining what this command does  (just a suggestion for consistency)
```suggestion

Download an object from a given bucket and region.
```

**for ex:**
Screen Shot 2022-03-22 at 11 38 05 AM

Comment (lamminade) [reply source/includes/aws/_objects.md:73]: also maybe one on line 52 since there's another missing

Comment (AnKhangC) [reply source/includes/aws/_objects.md:73]: [cf2c50c](https://github.com/cloudops/cloudmc-api-docs/pull/483/commits/cf2c50c99b01e7888ae0b3cf5bf5f0ddfe4d2cb9)

**Review by lamminade — APPROVED** [review]
thank you !!

**Pull request merged**